From bbd5d79f8c25ac21e05aa4ca1ba6a1fb60a4a1de Mon Sep 17 00:00:00 2001
From: Namit Solanki <nsolanki@codeaurora.org>
Date: Wed, 6 Mar 2019 17:59:28 +0530
Subject: [PATCH] libdrm: Include sysmacros from sys domain. * major, minor,
 makedev definitions have been moved to sysmacros.h * #include
 <sys/sysmacros.h> to refer to definitions.

Change-Id: I5983c576e0f65e60cdaded3245d4e6115490edbd
---
 libkms/linux.c | 1 +
 xf86drm.c      | 1 +
 2 files changed, 2 insertions(+)

diff --git a/libkms/linux.c b/libkms/linux.c
index 6e0da83..418d7d3 100644
--- a/libkms/linux.c
+++ b/libkms/linux.c
@@ -41,6 +41,7 @@
 #include <unistd.h>
 #include <sys/stat.h>
 #include <sys/types.h>
+#include <sys/sysmacros.h>
 #ifdef HAVE_SYS_MKDEV_H
 #include <sys/mkdev.h>
 #endif
diff --git a/xf86drm.c b/xf86drm.c
index 7e28b4f..ff81fbb 100644
--- a/xf86drm.c
+++ b/xf86drm.c
@@ -49,6 +49,7 @@
 #include <signal.h>
 #include <time.h>
 #include <sys/types.h>
+#include <sys/sysmacros.h>
 #include <sys/stat.h>
 #define stat_t struct stat
 #include <sys/ioctl.h>
-- 
1.9.1

